### PR TITLE
Fix dropdown menu on mobile devices

### LINKED ISSUE #33: mobile menus are not working
when viewing the site on the mobile, the submenus are not shown when clicking on the main menu items.

### PR DESCRIPTION
**Pull request by fabiopaiva**
Fixes #33

### CODE REVIEW

**Review by jkokkeler — CHANGES_REQUESTED** [review]
Check to see if you can fix those

File: layouts/partials/header.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
             {{ range site.Menus.main }}
             {{ if .HasChildren }}
             <li class="nav-item dropdown">
-              <a class="nav-link dropdown-toggle" href="{{ .URL | relLangURL }}">
+              <a class="nav-link dropdown-toggle" {{ if .URL }} href="{{ .URL | relLangURL }}" {{ end }}>
```
[the remark this answers is not in the document]
Comment (fabiopaiva) [reply layouts/partials/header.html:46]: It has to be clickable. On mobile devices, the tap will trigger the dropdown, not the `hover` event. 
This is how Bootstrap works. https://getbootstrap.com/docs/4.0/components/navs/#tabs-with-dropdowns
To improve it I could add some accessibility parameters
```
<a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Dropdown</a>
```

File: config.toml [hunk 2]
```diff
@@ -53,7 +53,6 @@ link = "plugins/scrollreveal/scrollreveal.min.js"
 # main menu
 [[menu.main]]
 name = "Solutions"
-URL = "solutions"
```
[the remark this answers is not in the document]
Comment (fabiopaiva) [reply config.toml:56]: what do you mean by `top`?

[the remark this answers is not in the document]
Comment (fabiopaiva) [reply config.toml:56]: ah, I think I understood it. You mean, a link to the top of the page. 
I think we could add a new dropdown menu item called `Our solutions`?

File: assets/scss/templates/_navigation.scss [hunk 3]
```diff
@@ -107,6 +107,7 @@ link:focus,
   .dropdown {
     &:hover {
       .dropdown-menu {
+        display: block;
```
[the remark this answers is not in the document]
Comment (fabiopaiva) [reply assets/scss/templates/_navigation.scss:110]: I couldn't reproduce it. It worked fine on my phone. 
Or did I misunderstand the problem?

https://user-images.githubusercontent.com/2126528/111760677-4690bb80-889f-11eb-803e-e6dfe4698c51.MOV

File: config.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jkokkeler) [reply config.toml:56]: Could be, check with Martin

File: layouts/partials/header.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkokkeler) [reply layouts/partials/header.html]: You could reset the pointer, that's what I was getting at

File: assets/scss/templates/_navigation.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (jkokkeler) [reply assets/scss/templates/_navigation.scss:110]: https://user-images.githubusercontent.com/1210221/112014463-342ab200-8b2b-11eb-81cb-b92107d6ce92.mp4

Please see attached, menu closes when selecting different menu items

[the remark this answers is not in the document]
Comment (fabiopaiva) [reply assets/scss/templates/_navigation.scss]: fixed

File: config.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (fabiopaiva) [reply config.toml:56]: He suggested without the top

**Review by jkokkeler — APPROVED** [review]

**Pull request merged**